File: webapi/Helpers/StatusEnum.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace webapi.Helpers
{
	public enum StatusEnum
	{
		/* Drink Status */
		[Display(Name = "New Order")] // In cart
		NewOrder = 1,
		[Display(Name = "In Progress")] // Order submitted
		InProgress = 2,
		[Display(Name = "Completed")]
		Completed = 3,
		[Display(Name = "Cancelled")]
		Cancelled = 4,
		/* Ingredient Status */
		[Display(Name = "Active")]
		Active = 5, // Default / Available
		[Display(Name = "Inactive")]
		Inactive = 6, // Unavailable at the moment
		[Display(Name = "Deleted")]
		Deleted = 7 // Hidden from the list
	}
}

File: angularapp/src/app/models/status.enum.ts
export enum Status {
  /* Ingredient Status */
  Active = 5, // Default / Available
  Inactive = 6, // Unavailable at the moment
  Deleted = 7 // Hidden from the list
}


export const statusOptions = [
  {value: Status.Active, viewValue: 'Available'},
  {value: Status.Inactive, viewValue: 'Unavailable'},
  {value: Status.Deleted, viewValue: 'Deleted'},
];

